### DISCUSSION TITLE
Autocomplete Dashboard tags when adding them

### DISCUSSION THREAD

**Discussion by Mario-Hofstaetter** [grafana/grafana#49919] [Dashboards, Panels & Visualizations]
Dashboard `tags` are a great way to categorize dashboads, filter [and search for them](https://grafana.com/docs/grafana/latest/dashboards/search/#filter-by-tags). When using the Dashboard List Panel or Dashboard Links, they are more or less mandatory to accomplish specific searches.

One has to be careful however not to misstype any tag names, including Case Sensitivity (❗ ).

It would be nice if the editor in dashboard settings supported autocomplete for existing tag names.

Example:
-> <https://play.grafana.org/dashboards/f/dFRSXVEiz/data-source-prometheus>

![grafik](https://user-images.githubusercontent.com/33002073/171179797-92d3bdd0-6673-4a93-a3c6-ec322843f073.png)

-> Edit <https://play.grafana.org/d/000000065/prometheus-console-tables?editview=settings>

Entering `prom...` should bring up an autocompletion suggestion for `prometheus`:

![firefox_Df0ef5hTRf](https://user-images.githubusercontent.com/33002073/171180283-e47dc0f0-56a5-44c1-9948-d1a134d940cc.gif)

(Not everybody is able to use advanced techniques like dashboards as code using jsonnet and what not, for multiple reasons. Our dashboards are hand crafted in Grafana and then committed to git afterwards)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
